managers/BaseDirectorService.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const _ = require('lodash');
+const utils = require('../broker/lib/utils');
+
+class BaseDirectorService {
+  constructor(plan) {
+    this.plan = plan;
+  }
+
+  get settings() {
+    return this.plan.manager.settings;
+  }
+
+  static parseDeploymentName(deploymentName, subnet) {
```

Comment:
duplicate [code](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/managers/backup-manager/BackupService.js#L46-L53)

Reply:
This method was removed from BackupService and moved to BaseDirectorService. @muralisc correct?

Reply:
@kamath-prasad its removed from [here](https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/321/files#diff-b35054ba7edff4ba9736b79267cb63a2L47)